colorDC1/jupyter_notebook_config.json

```diff
@@ -1,8 +1,8 @@
 {
   "NotebookApp": {
-    "allow_origin" : "*",
-  	"token":"",
-  	"password":"",
+    "allow_origin": "*",
+    "token": "",
+    "password": "",
```

Comment:
I used `jq` (https://stedolan.github.io/jq/) here to clean up the formatting a bit.